packages/forms/src/elements/search/search.tsx

```diff
@@ -0,0 +1,154 @@
+import React, { useState, useEffect } from 'react';
+import { Field, FieldRenderProps } from 'react-final-form';
+import { Flex } from '@tpr/core';
+import { StyledInputLabel, InputElementHeading } from '../elements';
+import { FieldProps, FieldExtraProps } from '../../renderFields';
+import AccessibilityHelper from '../accessibilityHelper';
+import Autocomplete from 'accessible-autocomplete/react';
+import { filterResults, formatItemDefault } from './filterResults';
+import { act } from 'react-dom/test-utils';
+import styles from './search.module.scss';
+
+interface SearchProps extends FieldRenderProps<string>, FieldExtraProps {
+	callback?: Function;
+	formatItem?: (item: any) => string;
+	getSelectedItem?: (item: any) => string;
+	keyValue: string;
+	notFoundMessage?: string;
+	optionsArray?: any[];
+	searchService?: (x: string) => Promise<any>;
+}
+
+type PanelVisibility = 'visible' | 'hidden' | 'complete';
+
+const Search: React.FC<SearchProps> = React.memo(
+	({
+		callback,
+		cfg,
+		formatItem = formatItemDefault,
+		getSelectedItem,
+		hint,
+		inputWidth = 10,
+		keyValue,
+		label,
+		meta,
+		name,
+		notFoundMessage = 'Your search criteria has no match',
```

Comment:
"criteria" is plural, "has" is singular, so as a minimal change this should be "Your search criteria have no match". I think "There are no matches for your search criteria" would be a little clearer though.

Reply:
default text changed to "There are no matches for your search criteria"